cli/index.ts

```diff
@@ -66,21 +66,15 @@ const includeDepsOption = {
   }
 };
 
-const defaultSchemaOption = {
-  name: "default-schema",
+const schemaSuffixOption: INamedOption<yargs.Options> = {
```

Comment:
should this have a check() function checking that the provided string only contains alphanumerics and underscores?